cmake/developer_package/plugins/plugins.cmake

```diff
@@ -111,6 +113,10 @@ function(ov_add_plugin)
             add_cpplint_target(${OV_PLUGIN_NAME}_cpplint FOR_TARGETS ${OV_PLUGIN_NAME} CUSTOM_FILTERS ${custom_filter})
         endif()
 
+        if (OV_PLUGIN_ADD_CLANG_TIDY)
+            set(CMAKE_CXX_CLANG_TIDY clang-tidy-${CLANG_TIDY_REQUIRED_VERSION})
```

Comment:
I think you need to use target property instead of global variable https://cmake.org/cmake/help/latest/prop_tgt/LANG_CLANG_TIDY.html#prop_tgt:%3CLANG%3E_CLANG_TIDY

Reply:
Right, corrected